app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class= "container-fluid singup">
```

Comment:
- [ ] OPTIONAL
Your app is functioning great, however, I think we can improve further on our design to look more closer to the [template design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), so as make our app a good fit for showcasing on a portfolio. Kindly bear in mind the typography, color alignment and spacing of objects.
- You can consider improving these pages:
  * Splash screen 
  * Sign up and log in pages
  * Home page (categories page)
  * Transactions page
  * "Add a new category" page
  * "Add a new transaction" page